docs/runbooks/VirtApiRESTErrorsBurst.md

```diff
@@ -1,36 +1,56 @@
 # VirtApiRESTErrorsBurst
+<!--Edited by jherrman, 17.10.2022-->
 
 ## Meaning
 
-Virt-api REST calls are failing in a very high rate.
-Alert will be fired when more than 80% of the REST calls failed in virt-api for the last 5 minutes.
+More than 80% of REST calls have failed in `virt-api` in the last 5 minutes.
 
 ## Impact
 
-Very high rate of failed REST calls to virt-api could lead to slow response / execution of API calls,
-and can even lead to API calls being completely dismissed.
+Very high rate of failed REST calls to `virt-api` may lead to slow response and execution of API calls, and potentially to API calls being completely dismissed.
```

Comment:
"Very high rate" > "A very high rate"

`virt-api` > the `virt-api` pods

"may" > might" (ISG)